blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -332,14 +333,38 @@ private static SetRollup deserializeV1SetRollup(CodedInputStream in) throws IOEx
         }
         return rollup;
     }
-    
-    private static void serializeTimer(TimerRollup rollup, byte[] buf) throws IOException {
+
+    private static void serializeV1Timer(TimerRollup rollup, byte[] buf) throws IOException {
         CodedOutputStream out = CodedOutputStream.newInstance(buf);
         timerRollupSize.update(buf.length);
         out.writeRawByte(Constants.VERSION_1_TIMER);
+
+        // sum, count, countps, avg, max, min, var
+        out.writeRawVarint64((long)rollup.getSum());
```

Comment:
it seems strange to refactor the methods and just pass a long sum into it. I'd rather pass all fields into it. Also if you have common methods shared between 2 methods of serialization, then if it changes for 1 version, we have to make sure that it is compatible with the other version too.